universal-login-react/src/ui/Onboarding/Onboarding.tsx

```diff
@@ -52,20 +36,28 @@ export const Onboarding = (props: OnboardingProps) => {
                   <div className="perspective">
                     <WalletSelector
                       sdk={props.sdk}
-                      onCreateClick={onCreateClick}
+                      onCreateClick={async (ensName) => {
```

Comment:
Minor thing, I think when function passed to a component is longer than one line it should be extracted